__tests__/app.test.js

```diff
@@ -443,4 +443,17 @@ describe("GET /api/articles (topic query)", () => {
       expect(msg).toBe("Oops, does not exist yet")
     })
   });
+});
+
+describe('GET /api/articles/:article_id (comment_count)', () => {
```

Comment:
This can and should be merged/nested with the previous describe for the base endpoint